support/cas-server-support-passwordless-ldap/src/test/java/org/apereo/cas/impl/account/LdapPasswordlessUserAccountStoreTests.java

```diff
@@ -55,5 +56,6 @@ public void verifyAction() throws Exception {
         assertTrue(user.isPresent());
         assertEquals("passwordlessuser@example.org", user.get().getEmail());
         assertEquals("123456789", user.get().getPhone());
+        assertEquals(true, user.get().getRequestPassword());
```

Comment:
How could this test ever work? Is there a setting called `request-password`? and how could the LDAP server return true for this in this test? 

Was this actually tested?